lib/screens/explore_screens/vote_screens/proposals/proposals_list.dart

```diff
@@ -79,8 +79,6 @@ class _ProposalsListState extends State<ProposalsList> with AutomaticKeepAliveCl
         },
         builder: (context, state) {
           switch (state.pageState) {
-            case PageState.initial:
```

Comment:
Hi Raul, should we then remove return ProposalDetailsState(
                                                         --->    pageState: PageState.initial,

Reply:
Mm this enum state needs an initial value both in UI it can be handled by the default switch case clause. At least you want draw something different for the initial state in UI. 
I started to remove in some UI places this instruction for that reason.
What do you think @gguijarro-c-chwy should we remove all 
`case: PageState.initial`
In favor the default switch case?

Reply:
lets talk about it in the next meeting